packages/api-gateway-service/webpack.common.js

```diff
@@ -0,0 +1,52 @@
+const nodeExternals = require('webpack-node-externals');
+const path = require('path');
+const fs = require('fs');
+const Dotenv = require('dotenv-webpack');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const NodemonPlugin = require('nodemon-webpack-plugin');
+
+let envPath;
+
+if (!process.env.NODE_ENV || process.env.NODE_ENV === 'local') {
+  envPath = '.env';
+} else if (process.env.NODE_ENV === 'test') {
+  envPath = 'e2e/.test.env';
```

Comment:
Are you sure this path works? I don't see any e2e directory in your branch.

Reply:
e2e directory and test cases are included

Reply:
But are they working with the test env variables? Anjnee had a problem where the tests weren't using the test env values.
I think you may need to remove this test env config from here, and put it in jest.config.js

Reply:
Yes its working fine i tested it.